src/ui/components/TestSuite/views/TestRecording/TestEventDetails.tsx

```diff
@@ -62,6 +63,14 @@ function UserActionEventDetails({
     variable
   );
 
+  const context = useMemo(
```

Comment:
Unfortunately this is necessary because `TimeStampedPoint` has a `point` property but `InspectableTimestampedPointContext` expects an `executionPoint` property.